kubernetes/linera-validator/scylla.values.yaml

```diff
@@ -0,0 +1,16 @@
+developerMode: true
+sysctls:
+  - "fs.aio-max-nr=4082080"
```

Comment:
Only issue is that I'm not sure I can do an `if` here to run just on `kind` (at least my editor said it was wrong 😅), and not sure we want this for `GCP` or not

Reply:
Yeah all good - we'll use a different value on GCP anyway :)